docs/magicc_flags.rst

```diff
@@ -0,0 +1,215 @@
+.. include:: _custom_rst_shortcuts.rst
+
+.. _`MAGICC flags`:
+
+MAGICC flags
+============
+
+Appropriately named, this section describes some of the magical flags contained in
+MAGICC and the gotchas which we have discovered so far. If you have an issue, please
+raise it in the `issue tracker <https://github.com/openclimatedata/pymagicc/issues>`_
+and we will do our best to assist you before documenting the solution here.
+
+Before moving on, note that Fortran is a case-insensitive language. Thus you shouldn't
+worry if you see a flag written as ``MAGICC_FLAG``, ``magicc_flag`` or ``MagICC_FlAg``.
+In Pymagicc, the ``f90nml`` package takes care of this case-insensitivity for us.
+
+
+Conventions
+-----------
+
+MAGICC's flags have a few repeating conventions which can make them easier to
+understand. However, sometimes the naming convention conflicts with the behaviour so
+it's worth double checking if in doubt. Here we summarise a few key conventions:
+
+- endings like ``0NO1SCALE2SHIFT``: in general, if these flags are set to ``0`` then they will do nothing, if set to ``1`` they will adjust the 'input' series to 'target' series by scaling (i.e. multiplying the whole series by a constant) and if set to ``2`` they will adjust the input series to the target series by shifting (i.e. adding/subtracting a constant to the whole series).
+- ``ABC_NAMES``: defines the variable names which belong to the group which will commonly be referred to with a prefix of ``ABC`` e.g. ``FGAS_NAMES``: defines the F-gases in MAGICC.
+- ``ABC_SWITCHFROMCONC2EMIS_YEAR``: the year in which to switch from being concentration driven to emissions driven for the gas/group of gases ``ABC`` e.g. ``CO2_SWITCHFROMCONC2EMIS_YEAR`` defines the year in which to switch from being |CO2| concentration driven to |CO2| emissions driven.
+
+
+Undocumented flags
+------------------
+
+In the MAGICC6 binary shipped with Pymagicc, all flags are contained in
+``MAGCFG_DEFAULTALL_69.CFG``. Hence they are in some sense 'documented' and at least it
+is obvious where to look in order to see what all the possible options are.
+
+In MAGICC7, this isn't always the case. Hence, here we provide a list of flags which
+are not in ``MAGCFG_DEFAULTALL.CFG`` but are nonetheless valid and useable.
+
+- ``FGAS_ADJSTFUTREMIS2PAST_0NO1SCALE``: adjust future F-gas emissions to past emissions.
+- ``MHALO_ADJSTFUTREMIS2PAST_0NO1SCALE``: adjust future Montreal Protocol halogen emissions to past emissions.
+
+
+Configuration flags
+-------------------
+
+When you run MAGICC, you have a series of ``.CFG`` files which set parameter values.
+The first two are always (and must be) ``MAGCFG_DEFAULTALL.CFG`` and
+``MAGCFG_USER.CFG``. *Exception:* in the compiled binary that is shipped with
+Pymagicc, the file it looks for is ``MAGCFG_DEFAULTALL_69.CFG``, not
```

Comment:
An aside, but does the 69 stand for Version 6.9?

Reply:
I believe so although of course it's never that clear